process-dataset/constants.js

```diff
@@ -13,7 +13,7 @@ const TEMP_LOCAL_CELL_FEATURE_JSON = "cell-feature-analysis.json";
 const FILE_INFO_KEYS = [
     CELL_ID_KEY,
     FOV_ID_KEY,
-    CELL_LINE_NAME_KEY,
+    "groupBy",
```

Comment:
when we do phase two of this work, this goes away entirely, right?

Reply:
or can this go away right now?  it's a whole data item per cell

Reply:
yeah, I was trying not to break too many things at once. What we actually need to is add cell line as one of the "measured features", since it's currently not there.

Reply:
I've now done this for the 1.4 and 2.1 datasets. @tanyasg is going to update the fish dataset tomorrow